include/ship/controller/controldevice/controller/ControllerButton.h

```diff
@@ -45,6 +41,8 @@ class ControllerButton {
     uint8_t mPortIndex;
     CONTROLLERBUTTONS_T mBitmask;
     std::unordered_map<std::string, std::shared_ptr<ControllerButtonMapping>> mButtonMappings;
+
+    std::unordered_map<CONTROLLERBUTTONS_T, std::string> mButtonNames;
```

Comment:
maybe this could live on controldeck or controller instead, but having controllerbutton "reach up" to get that feels less than ideal to me (but so does passing it down through a ton of constructors and keeping it on every button)

i guess the other possible refactor here would be to only keep the one string associated with the button on each, but my goal was to limit changes to logic as much as possible and focus purely on the separating lus/fast/ship aspects here

the other thing to consider is how this choice impacts the inputeditorwindow stuff, so it might be worth just trying to pull that work into this PR and letting that inform the final decision